docs/environment_setup.md

```diff
@@ -68,6 +68,10 @@ Use `flytectl` to start a demo Flyte cluster:
 flytectl demo start
 ```
 
+After this completes, be sure to export the Flyte config as it will be essential later. Run the command in the output that looks like this: 
+```{prompt} export FLYTECTL_CONFIG= ~/<pathTo>/.flyte/config-sandbox.yaml
```

Comment:
the command has to be present in the next line, otherwise it won't render:

```suggestion
```{prompt} bash
export FLYTECTL_CONFIG= ~/<pathTo>/.flyte/config-sandbox.yaml
```